.prow.yaml

```diff
@@ -566,6 +566,29 @@ presubmits:
               memory: 1Gi
               cpu: 500m
 
+  - name: pull-machine-controller-e2e-nutanix
+    optional: true
+    always_run: true
```

Comment:
I would remove this line here

Reply:
All the other private providers like kubevirt and vSphere are set to `always_run`. Shouldn't we keep testing Nutanix as much as possible? Running the test is basically "free" for us.